CHANGES.rst

```diff
@@ -1,16 +1,21 @@
 Changes
 =======
 
-Version master (UNRELEASED)
+Version 0.8.0 (2021-11-17)
 ---------------------------
 
 - Adds ``get_disk_usage`` utility function to calculate disk usage for a directory.
-- Adds Yadage workflow specification loading utilities.
+- Adds ``Yadage`` workflow specification loading utilities.
 - Adds workspace validation utilities.
 - Adds ``Snakemake`` workflow engine integration.
-- Centralises ``fs`` package dependency
+- Adds custom objects API instance to k8s client.
+- Adds available worklow engines configuration.
+- Adds environment variable to define time between job controller connection checks.
+- Adds cluster health status endpoint.
+- Adds OpenAPI specifications with respect to user quotas.
+- Fixes ``fs`` package dependency.
```

Comment:
Not sure if Centralises -> Fixes is a good replacement. I guess this line refers to this [commit](https://github.com/reanahub/reana-commons/commit/383aed55569a20b6acbc210379996bbe50c29a35) among others? Perhaps it's not even relevant to have here?

Reply:
I agree that it seems to be irrelevant, removed it